<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="5" passed="2" failed="3" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-09-16T15:16:05 IST" name="Master Suite" finished-at="2024-09-16T15:16:54 IST" duration-ms="48928">
    <groups>
      <group name="Master">
        <method signature="TC001_AccountRegistrationTest.verify_account_registration()[pri:0, instance:testCases.TC001_AccountRegistrationTest@2e6a5539]" name="verify_account_registration" class="testCases.TC001_AccountRegistrationTest"/>
        <method signature="TC002_LoginTest.verify_login()[pri:0, instance:testCases.TC002_LoginTest@740d2e78]" name="verify_login" class="testCases.TC002_LoginTest"/>
      </group> <!-- Master -->
      <group name="Sanity">
        <method signature="TC002_LoginTest.verify_login()[pri:0, instance:testCases.TC002_LoginTest@740d2e78]" name="verify_login" class="testCases.TC002_LoginTest"/>
      </group> <!-- Sanity -->
      <group name="Regression">
        <method signature="TC001_AccountRegistrationTest.verify_account_registration()[pri:0, instance:testCases.TC001_AccountRegistrationTest@2e6a5539]" name="verify_account_registration" class="testCases.TC001_AccountRegistrationTest"/>
      </group> <!-- Regression -->
    </groups>
    <test started-at="2024-09-16T15:16:05 IST" name="Test" finished-at="2024-09-16T15:16:54 IST" duration-ms="48928">
      <class name="testCases.TC002_LoginTest">
        <test-method is-config="true" signature="setup(java.lang.String,java.lang.String)[pri:0, instance:testCases.TC002_LoginTest@740d2e78]" started-at="2024-09-16T15:16:11 IST" name="setup" finished-at="2024-09-16T15:16:14 IST" duration-ms="3063" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Windows]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="verify_login()[pri:0, instance:testCases.TC002_LoginTest@740d2e78]" started-at="2024-09-16T15:16:14 IST" name="verify_login" finished-at="2024-09-16T15:16:15 IST" duration-ms="1336" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verify_login -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testCases.TC002_LoginTest@740d2e78]" started-at="2024-09-16T15:16:15 IST" name="tearDown" finished-at="2024-09-16T15:16:16 IST" duration-ms="84" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- testCases.TC002_LoginTest -->
      <class name="testCases.TC_005edit">
        <test-method is-config="true" signature="setup(java.lang.String,java.lang.String)[pri:0, instance:testCases.TC_005edit@1c481ff2]" started-at="2024-09-16T15:16:16 IST" name="setup" finished-at="2024-09-16T15:16:19 IST" duration-ms="3279" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Windows]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="Edit_Account_Details()[pri:0, instance:testCases.TC_005edit@1c481ff2]" started-at="2024-09-16T15:16:19 IST" name="Edit_Account_Details" finished-at="2024-09-16T15:16:22 IST" duration-ms="3082" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Success: Your account has been successfully updated. expected:<Success: Your account has been successfully updated.> but was:<Confirmation message mismatch>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Success: Your account has been successfully updated. expected:<Success: Your account has been successfully updated.> but was:<Confirmation message mismatch>
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:115)
at testCases.TC_005edit.Edit_Account_Details(TC_005edit.java:50)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Edit_Account_Details -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testCases.TC_005edit@1c481ff2]" started-at="2024-09-16T15:16:22 IST" name="tearDown" finished-at="2024-09-16T15:16:22 IST" duration-ms="87" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- testCases.TC_005edit -->
      <class name="testCases.TC001_AccountRegistrationTest">
        <test-method is-config="true" signature="setup(java.lang.String,java.lang.String)[pri:0, instance:testCases.TC001_AccountRegistrationTest@2e6a5539]" started-at="2024-09-16T15:16:05 IST" name="setup" finished-at="2024-09-16T15:16:09 IST" duration-ms="4543" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Windows]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="verify_account_registration()[pri:0, instance:testCases.TC001_AccountRegistrationTest@2e6a5539]" started-at="2024-09-16T15:16:09 IST" name="verify_account_registration" finished-at="2024-09-16T15:16:11 IST" duration-ms="1641" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Your Account Has Been Created! expected:<Your Account Has Been Created!> but was:<Confirmation message mismatch>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Your Account Has Been Created! expected:<Your Account Has Been Created!> but was:<Confirmation message mismatch>
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:115)
at testCases.TC001_AccountRegistrationTest.verify_account_registration(TC001_AccountRegistrationTest.java:44)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verify_account_registration -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testCases.TC001_AccountRegistrationTest@2e6a5539]" started-at="2024-09-16T15:16:11 IST" name="tearDown" finished-at="2024-09-16T15:16:11 IST" duration-ms="89" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- testCases.TC001_AccountRegistrationTest -->
      <class name="testCases.TC007_component">
        <test-method is-config="true" signature="setup(java.lang.String,java.lang.String)[pri:0, instance:testCases.TC007_component@1b955cac]" started-at="2024-09-16T15:16:46 IST" name="setup" finished-at="2024-09-16T15:16:49 IST" duration-ms="3062" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Windows]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="component()[pri:0, instance:testCases.TC007_component@1b955cac]" started-at="2024-09-16T15:16:49 IST" name="component" finished-at="2024-09-16T15:16:53 IST" duration-ms="4370" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- component -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testCases.TC007_component@1b955cac]" started-at="2024-09-16T15:16:53 IST" name="tearDown" finished-at="2024-09-16T15:16:54 IST" duration-ms="85" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- testCases.TC007_component -->
      <class name="testCases.TC006_AddtocartTest">
        <test-method is-config="true" signature="setup(java.lang.String,java.lang.String)[pri:0, instance:testCases.TC006_AddtocartTest@72437d8d]" started-at="2024-09-16T15:16:22 IST" name="setup" finished-at="2024-09-16T15:16:25 IST" duration-ms="2984" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Windows]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="addingtocart()[pri:0, instance:testCases.TC006_AddtocartTest@72437d8d]" started-at="2024-09-16T15:16:25 IST" name="addingtocart" finished-at="2024-09-16T15:16:46 IST" duration-ms="20684" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Products marked with *** are not available in the desired quantity or not in stock!] but found [Products marked with *** are not available in the desired quantity or not in stock!
×]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Products marked with *** are not available in the desired quantity or not in stock!] but found [Products marked with *** are not available in the desired quantity or not in stock!
×]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at testCases.TC006_AddtocartTest.addingtocart(TC006_AddtocartTest.java:50)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addingtocart -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testCases.TC006_AddtocartTest@72437d8d]" started-at="2024-09-16T15:16:46 IST" name="tearDown" finished-at="2024-09-16T15:16:46 IST" duration-ms="93" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- testCases.TC006_AddtocartTest -->
    </test> <!-- Test -->
  </suite> <!-- Master Suite -->
</testng-results>
